wlauto/common/android/device.py

```diff
@@ -589,7 +589,7 @@ def perform_unlock_swipe(self):
             start = 100
             stop = width - start
             self.execute(command.format(start, swipe_heigh, stop, swipe_heigh))
-        if self.swipe_to_unlock == "vertical":
```

Comment:
This isn't part of the same change.  As you said, it is device specific too so best remove it.